package.json

```diff
@@ -43,5 +40,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
@parthvaghani overall, it's a good start, but you should really test it - It still jumps, cube is disappering or goes black upon remove a map